smartsim/settings/arguments/batch/slurm.py

```diff
@@ -56,7 +56,11 @@ def set_walltime(self, walltime: str) -> None:
         format = "HH:MM:SS"
 
```

Comment:
Could you add this to slurm.set:

if key in self._batch_args and key != self._batch_args[key]:
            logger.warning(f"Overwriting argument '{key}' with value '{value}'")